src/app/constructor/dashboard/dashboard-constructor.template.html

```diff
@@ -13,17 +13,16 @@
 </div>
 
 <sh-dashboard-view
-        [widgets]="widgets"
-        (onRemoveWidget)="onRemoveWidget($event)">
+        [widgets]="selectedDevices"
+        (onRemoveWidget)="onRemoveSelectedDevice($event)">
 </sh-dashboard-view>
 
 <div class="step">
     <span>Step 5: Add widgets on dashboard</span>
 </div>
 
 <sh-sensors-list
-        [sensors]="sensors"
-        [mappedSensors]="widgets"
-        (onAddSensor)="onAddSensor($event)"
-        (onRemoveSensor)="onRemoveSensor($event)">
+        [mappedSensors]="selectedDevices"
```

Comment:
Devices it is an essence which is more wider than the sensors. Devices contain all sensors but  sensors don't contain all devices.